nest-cli/src/commands/mod.rs

```diff
@@ -26,11 +28,13 @@ pub fn print_transactions(transactions: &[Transaction]) {
             "{}",
             match transaction {
                 Transaction::Pull(p) => {
-                    format!("{} {}", "pull".cyan(), p.target_repository().name()).bold()
+                    format!("{:>8.8} {}", "pull".cyan(), p.target_repository().name()).bold()
                 }
-                Transaction::Install(_) => "install".green().bold(),
-                Transaction::Remove(_) => "remove".red().bold(),
-                Transaction::Upgrade(_) => "upgrade".yellow().bold(),
+                Transaction::Install(i) => {
+                    format!("{:>8.8} {}", "install".green(), i.target()).bold()
+                }
+                Transaction::Remove(r) => format!("{:>8.8} {}", "remove".red(), r.target()).bold(),
+                Transaction::Upgrade(_) => format!("{:>8.8}", "upgrade".yellow()).bold(),
```

Comment:
The `upgrade` doesn't display the target's name? Is that intended, because you haven't finished to handle upgrades yet?

Reply:
Yep, I will add proper display (name + old and new versions) only after implementing the upgrade operation